DeepSpeech.py

```diff
@@ -597,57 +573,38 @@ def get_git_branch():
 # Helpers
 # =======
 
-def calculate_and_print_wer_report(caption, results_tuple):
-    r"""
-    This routine will print a WER report with a given caption.
-    It'll print the `mean` WER plus summaries of the ``ds_report_count`` top lowest
-    loss items from the provided WER results tuple
-    (only items with WER!=0 and ordered by their WER).
-    """
-    items = list(zip(*results_tuple))
-
-    count = len(items)
+def calculate_report(results_tuple):
+    r'''
+    This routine will calculate a WER report.
+    It'll compute the `mean` WER and create ``Sample`` objects of the ``report_count`` top lowest
+    loss items from the provided WER results tuple (only items with WER!=0 and ordered by their WER).
+    '''
+    samples = []
+    items = zip(*results_tuple)
```

Comment:
For Python 2/3 compat, this needs to be `list(zip(...`, and then we need to import `six.moves.zip`.